src/safariservices.cs

```diff
@@ -156,6 +156,7 @@ interface SFSafariPage
 
 	[Mac (10,12)]
 	[Protocol]
+	[Model]
```

Comment:
I'd rather tweak the test than shipping extra code that's unlikely to be ever used. Otherwise the rest all looks good to me!